en/asgardeo/docs/references/service-extensions/pre-flow-extensions/pre-update-profile-action/api/pre-update-profile-action-v1.yaml

```diff
@@ -0,0 +1,296 @@
+openapi: 3.0.1
+info:
+  title: API contract for pre update profile action
+  description: This API defines the REST API contract for a service that implements logic to extend the profile update flow of Asgardeo.
```

Comment:
This API specifies the REST contract for a service that extends the profile update flow in Asgardeo.